### PR TITLE
spec: clarify referrers response for index

### PR DESCRIPTION
**Pull request by sajayantony**
Addresses : #458 
@sudo-bmitch

### CODE REVIEW

**Review by jdolitsky — DISMISSED** [review]

File: spec.md [hunk 1]
```diff
@@ -556,10 +556,11 @@ If the request is invalid, such as a `<digest>` with an invalid syntax, a `400 B
 
 Upon success, the response MUST be a JSON body with an image index containing a list of descriptors.
 The `Content-Type` header MUST be set to `application/vnd.oci.image.index.v1+json`.
-Each descriptor is of an image manifest in the same `<name>` namespace with a `subject` field that specifies the value of `<digest>`.
-The descriptors MUST include an `artifactType` field that is set to the value of the `artifactType` in the image manifest, if present.
+Each descriptor is of an image manifest or index in the same `<name>` namespace with a `subject` field that specifies the value of `<digest>`.
+The descriptors MUST include an `artifactType` field that is set to the value of the `artifactType` in the image manifest or index, if present.
 If the `artifactType` is empty or missing in the image manifest, the value of `artifactType` MUST be set to the config descriptor `mediaType` value.
-The descriptors MUST include annotations from the image manifest.
+If the `artifactType` is empty or missing in an index, the `arfactType` MUST be omitted.
```
Comment (sudo-bmitch) [inline spec.md:562]:
```suggestion
If the `artifactType` is empty or missing in an index, the `artifactType` MUST be omitted.
```

**Review by sudo-bmitch — CHANGES_REQUESTED** [review]
Just one typo nit. Otherwise LGTM.

**Comment by sajayantony** [comment]
Thanks @sudo-bmitch @jdolitsky PTAL.

**Review by sudo-bmitch — APPROVED** [review]
LGTM

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**